custom-fonts.md

```diff
@@ -23,6 +41,72 @@ flutter:
         - asset: assets/fonts/abrilfatface/AbrilFatface-Regular.ttf
 ```
 
+### Using the font
+
+Second use the font by delcaring the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html)
+as a constant. 
+Then set the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html) `fontFamily` property 
+and use the font family specified in the `pubsec.yaml` delcarations. 
+The font family `pubsec.yaml` declaration will look something like `family: [declared-font-family]`.
```

Comment:
Oh, I was suggesting to actually include the `family: ...` into the example pubspec.yaml you have farther up in the document. Then, here you say "The `fontFamily` property must match the `family` name from `pubspec.yaml`.